docs/workforce-management-framework/topics/ref-passportauth-securitystrategy.adoc

```diff
@@ -30,19 +30,17 @@ app.get('/testAdminEndpoint', authService.protect('admin'), (req: express.Reques
 });
 ----
 
-NOTE: Demo roles are defined as an array in link:{WFM-RC-CoreURL}{WFM-RC-Branch}/demo/server/src/modules/wfm-user/users.json[user.json].
+NOTE: Demo roles are defined in link:{WFM-RC-CoreURL}{WFM-RC-Branch}/demo/server/src/modules/wfm-user/users.json[user.json].
 
 [id='defining-password-storage-{chapter}']
 == Defining Password Storage
-The link:../../../api/{WFM-RC-Api-Version}{WFM-RC-Api-User-Repository}#getuserbylogin[User Repository Interface] 
-is used to store *PassportAuth* user data.
-To point *PassportAuth* to a datasource, you must implement the User Repository Interface.
+The *PassportAuth* user data is stored in link:../../../api/{WFM-RC-Api-Version}{WFM-RC-Api-User-Repository}#getuserbylogin[User Repository Interface].
+User Repository Interface must be implemented in order to point *PassportAuth* to a datasource,
```

Comment:
"Make sure that the _User Repository Interface_ is implemented..."

I only learned this sentence structure this week as a result of a peer review - the docs style guide states that we should avoid using the word "must" and "have to".